sentences/en/light_HassTurnOff.yaml

```diff
@@ -39,25 +33,25 @@ intents:
           - "<floor> <light> (off|out)"
           - "<light> (off|out) [<in>] <floor>"
           - "<turn> (off|out) [<all>] <floor> <light>"
-          - "<turn> (off|out) [<all>] <light> [<in>] <floor>"
+          - "<turn> (off|out) [<all>] [<the>] <light> [<in>] <floor>"
```

Comment:
`<all>` is already `(all[[ of] <the>]|every[ single]|each[ and every])`. Why would you add `[<the>]` as an extra? You could also get a match for `every the light`, which you don't want

Reply:
Ah. This is because I wanted it to match in cases <all> was omitted. I'll change to `[(<all>|<the>)]`.